app/vmui/packages/vmui/src/pages/ExploreAnomaly/ExploreAnomaly.tsx

```diff
@@ -17,6 +17,16 @@ import { useFetchAnomalySeries } from "./hooks/useFetchAnomalySeries";
 import { useQueryDispatch } from "../../state/query/QueryStateContext";
 import { useTimeDispatch } from "../../state/time/TimeStateContext";
 
+const anomalySeries = [
+  ForecastType.yhat,
+  ForecastType.yhatUpper,
+  ForecastType.yhatLower,
+  ForecastType.anomalyScore
+];
+
+// maybe get from backend from config.yml?
+const ANOMALY_SCORE_THRESHOLD = 1;
```

Comment:
The config.yml doesn't contain anything about 1.0 threshold. vmanomaly just produces the anomaly_score in such a way to satisfy models' predictions if it's < 1.0. It's up to alerting or GUI to actually cut off at 1.0 threshold. We can even make a UI field or slider to move threshold up or down. 
For now I think it's OK to hardcode as it is.

Reply:
Thank you for the clarification. I have updated the comment.